### PR TITLE
DOCS-1980 Add IM getting started

### PR DESCRIPTION
**Pull request by maycmlee**
### What does this PR do?
Adding IM Getting Started

### Motivation
DOCS-1980

### Preview

https://docs-staging.datadoghq.com/may-lee/im-getting-started2/getting_started/incident_management

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

**Comment by kayayarai** [comment]
Oops! We remembered and then forgot we need to update the main sidebar nav. The change goes in the file `/config/_default/menus/menus.en.yaml` somewhere around line 337. Follow the pattern and update any weight values as needed.

File: content/en/getting_started/incident_management/_index.md [hunk 1]
```diff
@@ -0,0 +1,282 @@
...
+
+## Overview
+
+Datadog Incident Management is for tracking and communicating about an issue you've identified with your metrics, traces, or logs.
+
+This guide walks you through using the Datadog app for declaring an incident, updating the incident as investigation and remediation progresses, and generating a postmortem when the incident has been resolved. The example assumes the [Slack integration][1] is enabled.
+
+## Walking through an incident from issue detection to resolution
+
+### Declaring an Incident
```
Comment (kayayarai) [inline content/en/getting_started/incident_management/_index.md:37]:
```suggestion
### Declaring an incident
```

File: content/en/getting_started/incident_management/_index.md [hunk 2]
```diff
@@ -0,0 +1,282 @@
...
+
+**Scenario:** A monitor is alerting on a high number of errors which may be slowing down several services. It's unclear whether customers are being impacted.
+
+For this guide, we are using the [Datadog Clipboard][2] to declare an incident.
```
Comment (kayayarai) [inline content/en/getting_started/incident_management/_index.md:41]:
```suggestion
This guide describes using the [Datadog Clipboard][2] to declare an incident.
```

File: content/en/getting_started/incident_management/_index.md [hunk 3]
```diff
@@ -0,0 +1,282 @@
...
+
+6. Add yourself to _Add recipients_.
+
+7. Click **Send**.
+
+    You should receive an email with the message.
+
+    You can create customized templates by clicking on **Manage Templates** > **New Template**. Group templates together using the _Category_ field.
+
+### Resolution / Postmortem
```
Comment (kayayarai) [inline content/en/getting_started/incident_management/_index.md:228]:
```suggestion
### Resolution and postmortem
```

File: content/en/monitors/incident_management/_index.md [hunk 4]
```diff
@@ -117,75 +121,49 @@ The default includes the statuses **Active**, **Stable**, and **Resolved**. **Co
...
-Create mitigation or post-incident remediation tasks. You can track any of your tasks here by adding tasks in the text field, setting a due date, and assigning a team member. You can complete them by checking off the box when they are done.
+- [PagerDuty][8] to send incident notifications to PagerDuty.
+- [Jira][9] to create a Jira ticket for an incident.
+- [Webhooks][10] to send incident notifications using webhooks (for example, [sending SMS to Twilio][11]).
 
-Link to a postmortem document, look back on exactly what went wrong, and add follow-up tasks. Postmortems created in Datadog [Notebooks][8] support live collaboration; to link to an existing notebook, click the plus under `Other Docs`. Click the linked notebook to edit with teammates in real-time.
+## Ready to try it out?
 
-{{< img src="monitors/incidents/workflow-5-postmortem-1.png" alt="Postmortem"  style="width:60%;">}}
+Work through an example workflow in our [Getting Started with Incident Management][12].
```
Comment (kayayarai) [inline content/en/monitors/incident_management/_index.md:156]:
```suggestion
Work through an example workflow in the [Getting Started with Incident Management][12] guide.
```

File: content/en/monitors/incident_management/_index.md [hunk 5]
```diff
@@ -117,75 +121,49 @@ The default includes the statuses **Active**, **Stable**, and **Resolved**. **Co
...
 
 [1]: https://app.datadoghq.com/incidents
 [2]: https://app.datadoghq.com/incidents/settings
 [3]: /integrations/slack/?tab=slackapplicationbeta#using-the-slack-app
 [4]: https://docs.datadoghq.com/integrations/slack/?tab=slackapplicationus
```
Comment (kayayarai) [inline content/en/monitors/incident_management/_index.md:161]:
```suggestion
[4]: /integrations/slack/
```

File: content/en/monitors/incident_management/_index.md [hunk 6]
```diff
@@ -117,75 +121,49 @@ The default includes the statuses **Active**, **Stable**, and **Resolved**. **Co
...
-[6]: /tracing/#2-instrument-your-application
-[7]: https://app.datadoghq.com/incidents/ddslackapp
-[8]: https://app.datadoghq.com/notebook/list
+[5]: https://app.datadoghq.com/incidents/ddslackapp
+[6]: /getting_started/tagging/assigning_tags?tab=noncontainerizedenvironments#overview
+[7]: /tracing/#2-instrument-your-application
+[8]: https://docs.datadoghq.com/integrations/pagerduty/#overview
+[9]: https://docs.datadoghq.com/integrations/jira/#overview
+[10]: https://docs.datadoghq.com/integrations/webhooks/#overview
+[11]: https://docs.datadoghq.com/integrations/webhooks/#sending-sms-through-twilio
```
Comment (kayayarai) [inline content/en/monitors/incident_management/_index.md:168]:
```suggestion
[8]: /integrations/pagerduty/
[9]: /integrations/jira/
[10]: /integrations/webhooks/
[11]: /integrations/webhooks/#sending-sms-through-twilio
```

File: content/en/getting_started/incident_management/_index.md [hunk 7]
```diff
@@ -0,0 +1,282 @@
...
+
+{{< partial name="whats-next/whats-next.html" >}}
+
+[1]:/integrations/slack/?tab=slackapplicationus#overview
+[2]:/monitors/incident_management/datadog_clipboard
+[3]:/notebooks/#overview
+[4]:/monitors/incident_management/#from-a-graph
+[5]:/monitors/incident_management/#from-a-monitor
+[6]:/api/latest/incidents/#create-an-incident
+[7]:/incident_management/#status-levels
```
Comment (kayayarai) [inline content/en/getting_started/incident_management/_index.md:280]:
```suggestion
[7]: /monitors/incident_management/#status-levels
```

File: content/en/getting_started/incident_management/_index.md [hunk 8]
```diff
@@ -0,0 +1,282 @@
...
+
+To customize Incident Management, go to the [incident settings page][9]. From the Datadog menu on the left-hand side, go to **Monitors** > **Incidents** (if you get an Incident Management welcome screen, click on **Get Started**). Then on the top right corner, click **Settings**.
+
+{{< img src="getting_started/incident_management/im_settings_button.png" alt="Settings" responsive="true" style="width:100%;">}} 
+
+## Further Reading
+
+{{< partial name="whats-next/whats-next.html" >}}
+
+[1]:/integrations/slack/?tab=slackapplicationus#overview
```
Comment (kayayarai) [inline content/en/getting_started/incident_management/_index.md:274]:
```suggestion
[1]:/integrations/slack/
```

File: content/en/getting_started/incident_management/_index.md [hunk 9]
```diff
@@ -0,0 +1,282 @@
...
+
+## Walking through an incident from issue detection to resolution
+
+### Declaring an Incident
+
+**Scenario:** A monitor is alerting on a high number of errors which may be slowing down several services. It's unclear whether customers are being impacted.
+
+For this guide, we are using the [Datadog Clipboard][2] to declare an incident.
+
+1. Open the Clipboard: **Ctrl**/**Cmd** + **Shift** + **K**
```
Comment (kayayarai) [inline content/en/getting_started/incident_management/_index.md:43]:
```suggestion
1. Open the Clipboard: **Ctrl**/**Cmd** + **Shift** + **K**.
```

File: content/en/getting_started/incident_management/_index.md [hunk 10]
```diff
@@ -0,0 +1,282 @@
...
+
+**Scenario:** The issue has been mitigated, and the team is monitoring the situation. The incident status is now stable.
+
+In the _Communications_ section, you can send out a notification updating the status of the incident.
+
+1. Navigate back to the _Overview_ section.
+
+2. Click **Edit** in the _Properties_ box and change the status to _stable_.
+
+3. Click **Save.**
```
Comment (kayayarai) [inline content/en/getting_started/incident_management/_index.md:210]:
```suggestion
3. Click **Save**.
```

File: content/en/getting_started/incident_management/_index.md [hunk 11]
```diff
@@ -0,0 +1,282 @@
...
+
+5. Change the status to _resolved_.
+
+6. Click **Save**.
+
+    When an incident's status is set to resolved, a _Generate Postmortem_ button appears at the top.
+
+    {{< img src="getting_started/incident_management/generate_postmortem.png" alt="Generate Postmortem" responsive="true" style="width:80%;">}}
+
+7. Click **Generate Postmortem.**
```
Comment (kayayarai) [inline content/en/getting_started/incident_management/_index.md:250]:
```suggestion
7. Click **Generate Postmortem**.
```

File: content/en/getting_started/incident_management/_index.md [hunk 12]
```diff
@@ -0,0 +1,282 @@
...
+
+8. For the timeline section, select **Marked as Important** so that only the _Important_ events are added to the postmortem.
+
+9. Click **Generate.**
```
Comment (kayayarai) [inline content/en/getting_started/incident_management/_index.md:254]:
```suggestion
9. Click **Generate**.
```

File: content/en/getting_started/incident_management/_index.md [hunk 13]
```diff
@@ -0,0 +1,282 @@
...
+
+    If there are follow-up tasks that you and your team need to complete to ensure the issue doesn't happen again, add those and track them in the Remediation's _Incident Tasks_ section.
+
+## Customizing your incident management workflow
+
+Datadog Incident Management can be customized with different severity and status levels, based on your organization's needs, and also include additional information such as APM services and teams related to the incident. For more information, see this [section][7] of the Incident Management page.
+
+You can also set up notification rules to automatically notify specific people or services based on an incident's severity level. For more information, see the [Notification Rules][8] documentation.
+
+To customize Incident Management, go to the [incident settings page][9]. From the Datadog menu on the left-hand side, go to **Monitors** > **Incidents** (if you get an Incident Management welcome screen, click on **Get Started**). Then on the top right corner, click **Settings**.
```
Comment (kayayarai) [inline content/en/getting_started/incident_management/_index.md:266]:
```suggestion
To customize Incident Management, go to the [incident settings page][9]. From the Datadog menu on the left-hand side, go to **Monitors** > **Incidents** (if you get an Incident Management welcome screen, click **Get Started**). Then on the top right corner, click **Settings**.
```

**Review by kayayarai — CHANGES_REQUESTED** [review]
A few edits, plus add entry to main sidebar nav. 🎉

File: content/en/getting_started/incident_management/_index.md (hunk 3 above)
Comment (maycmlee) [reply content/en/getting_started/incident_management/_index.md:228]: Should postmortem be capitalized?

Comment (maycmlee) [reply content/en/getting_started/incident_management/_index.md:228]: nevermind!

**Review by kayayarai — APPROVED** [review]

**Review by apigirl — APPROVED** [review]

**Pull request merged**